SCClassLibrary/Common/Control/Server.sc

```diff
@@ -727,7 +781,7 @@ Server {
 
 	// defaultGroups for all clients on this server:
 
-	allClientIDs { ^(0..options.maxLogins-1) }
+	allClientIDs { ^(0..this.maxNumClients-1) }
```

Comment:
if `maxNumClients` is not yet initialized, this will throw an unspecific error.

Reply:
when maxNumClients is nil, it falls back on options.maxLogins:
```	maxNumClients { ^maxNumClients ?? { options.maxLogins } }```
So unless users intentionally set options.maxLogins to nil, this should not fail;
if they do that, it probably should fail, no?

Reply:
as it is a public method it should at least throw an error that informs the caller about the reason.

Reply:
ok!